### PR TITLE
ESI-13735 Re-sync resources CLI extension

### PR DESCRIPTION
**Pull request by marcin-ptaszynski**
Resync command takes server config and based on it, resyncs all
syncable resources to sync backend.

Especially useful for recreating environment on fresh etcd cluster

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -309,6 +308,52 @@ Useful for development purposes.`,
...
+			}
+			if err := os.Chdir(path.Dir(configFile)); err != nil {
+				panic(fmt.Errorf("Chdir error: %s", err))
+			}
+
+			dbConn, err := db.CreateFromConfig(config)
+			sync, err := sync_util.CreateFromConfig(config)
+			if err != nil {
+				log.Error("Failed to create sync, err: %s", err)
+
```
Comment (morrisson) [inline cli/cli.go]: Should gohan exit if sync is not available?

File: server/sync_writer.go [hunk 2]
```diff
@@ -113,18 +113,19 @@ func (writer *SyncWriter) Run(ctx context.Context) error {
 // Sync runs a synchronization iteration, which
 // executes requests in the event table.
 // todo: make private once tests are fixed
```
Comment (morrisson) [inline server/sync_writer.go]: nit: this todo line should be deleted now Sync is public.

File: cli/cli.go (hunk 1 above)
Comment (marcin-ptaszynski) [inline cli/cli.go]: done, using log.Fatal(f) now, which exits.

File: server/sync_writer.go (hunk 2 above)
Comment (marcin-ptaszynski) [inline server/sync_writer.go]: done

**Pull request merged**